### DISCUSSION TITLE
The first Locha Mesh's Turpial dev-kits in the wild | 2021-01-03

### DISCUSSION THREAD

**Discussion by rdymac** [btcven/locha#175] [Show and tell]
## The first Locha Mesh's Turpial dev-kits in the wild | 2021-01-03
![20210103_144119 2](https://user-images.githubusercontent.com/2995401/104130264-01687e80-5370-11eb-95f2-a0ec56f11345.jpg)

On January 3rd 2021, Luis and Jean from the Locha Mesh team deployed the first Turpial dev-kit nodes to carry out range tests with static and mobile nodes from different locations, with nodes in line of sight and NLoS. The experiment was designed with default values, 200 kbps speed, and omnidirectional antennas.

Initially, a static node was deployed in the coordinates 40.398935, -4.503101, its purpose was to send UDP packets periodically to measure the quality of the signal in the chosen test points by using an SDR, while additionally verifying the packet reception with a mobile Turpial node that was with the team the whole time.

![20210103_143525 2](https://user-images.githubusercontent.com/2995401/104130893-8bfead00-5373-11eb-8240-8405137c33c1.jpg)

Once the on site Turpial node was up and running, our team left for a drive to the first checkpoint, measuring the quality of the connection during the ride as the reception of the UDP packets broadcasted by the base node.

The team witnessed and recorded the outstanding quality of the reception in several points at different altitudes, inside and around the town.

First checkpoint (A) was located at 40.401402, -4.481335, 1.8 km from base node in LoS, so the values obtained were the ones we expected, good RSSI and 100% of sent packets were captured.

Second checkpoint (B) at 40.430277, -4.477927, 4.13 km from base node NLoS, the team recorded excellent results.

![20210103_152133 2](https://user-images.githubusercontent.com/2995401/104181589-60c99b80-540f-11eb-8fd3-cbd9f3f80dcc.jpg)

The team then headed to the third checkpoint (C) located at 40.458331, -4.480083 where during the ride several shadow and coverage zones were identified. Nevertheless as the team arrived at the checkpoint NLoS, they noticed a stable signal being received with a RSSI of -103 dBm and 70% of the UDP packets casted by the base node were captured.  

Summarizing, the largest distance reached in this test was 7.16 km NLoS in the third checkpoint.

![January-3rd-LochaMesh](https://user-images.githubusercontent.com/2995401/104176615-d6ca0480-5407-11eb-8a94-5385f8943190.png)

We want to thank the community for supporting us, this has been an exciting experience since we started. The Turpial development kits will be ready to ship as early as Q1 to all of our early supporters so they can test them and start creating a true resilient, decentralized, censorship-resistant mesh network.

Kudos to Luis, Jean and the rest of the team. Keep up the good work!

Follow us on Twitter [@Locha_io](https://twitter.com/Locha_io), subscribe to [the mailing list](http://locha.io/signup), and pre-order the Turpial dev-kits here: https://locha.io/#pre-order

 
![20210103_155221 2](https://user-images.githubusercontent.com/2995401/104181786-a8502780-540f-11eb-9672-b0ccc8e2952a.jpg)

**Comment by gustavosinbandera1** [comment]
Good Job Guys, really all of these things involved in the Turpial device are fantastic, I hope this project can be a communications revolution :+1:
